src/src/components/form/FormErrors.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+    <div class="form-error-container">
+        <span class="message" v-show="messageList.length">
+            <p>
+                <fa icon="exclamation-triangle"></fa>
+                <span
+                  v-for="(message, index) in messageList"
+                  v-html="message.message"
+                  :key="index"></span>
+            </p>
+        </span>
+        <span class="error" v-show="errorList.length">
+            <p>
+                <fa icon="exclamation-triangle"></fa>
+                <span
+                v-for="(error,index) in errorList"
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 10 spaces but found 16 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
          v-for="(error,index) in errorList"
```